include/micm/util/property_keys.hpp

```diff
@@ -4,5 +4,11 @@
 
 #include <string>
 
-static const std::string GAS_DIFFUSION_COEFFICIENT = "diffusion coefficient [m2 s-1]";
-static const std::string MOLECULAR_WEIGHT = "molecular weight [kg mol-1]";
+namespace micm
+{
+  namespace keys
+  {
+    static const std::string GAS_DIFFUSION_COEFFICIENT = "diffusion coefficient [m2 s-1]";
+    static const std::string MOLECULAR_WEIGHT = "molecular weight [kg mol-1]";
```

Comment:
Could the namespace be `property_keys` or something more specific? I think `keys` maybe too general.

Reply:
agreed. changed!